content/docs/forms.md

```diff
@@ -9,23 +9,24 @@ redirect_from:
   - "docs/forms-zh-CN.html"
 ---
 
-HTML form elements work a little bit differently from other DOM elements in React, because form elements naturally keep some internal state. For example, this form in plain HTML accepts a single name:
+रियेक्ट मै HTML फॉर्म के तत्व DOM के तत्वो से थोड़ा भिन्न तरीके से काम करते है, क्योंकि फार्म के तत्व स्वाभाविक रूप से कुछ आंतरिक स्थिति रखते हैं, जैसे की, सादे HTML में यह फॉर्म एक ही नाम स्वीकार करेगा:
 
 ```html
 <form>
   <label>
-    Name:
-    <input type="text" name="name" />
+    Name
+    <input  type ="text" name ="name" />
   </label>
-  <input type="submit" value="Submit" />
+  <input  type ="submit" value  ="submit" />
 </form>
 ```
 
-This form has the default HTML form behavior of browsing to a new page when the user submits the form. If you want this behavior in React, it just works. But in most cases, it's convenient to have a JavaScript function that handles the submission of the form and has access to the data that the user entered into the form. The standard way to achieve this is with a technique called "controlled components".
 
-## Controlled Components {#controlled-components}
+जब उपयोगकर्ता फॉर्म को सबमिट करता है, तो इस फ़ॉर्म में नए पेज पर ब्राउज़िंग का डिफ़ॉल्ट HTML फॉर्म व्यवहार होता है। यदि आप इस व्यवहार को प्रतिक्रिया में चाहते हैं, तो यह काम करता है। लेकिन ज्यादातर मामलों में, जावास्क्रिप्ट फ़ंक्शन का उपयोग करना सुविधाजनक होता है जो फ़ॉर्म को प्रस्तुत करने में सक्षम होता है और उस डेटा तक पहुंच होती है जो उपयोगकर्ता ने फॉर्म में दर्ज किया था। इसे प्राप्त करने का मानक तरीका "नियंत्रित घटकों" नामक एक तकनीक के साथ है।
```

Comment:
`तो इस फ़ॉर्म में नए पेज पर ब्राउज़िंग का डिफ़ॉल्ट HTML फॉर्म व्यवहार होता है।`

does not makes any sense, can rephrase it to a meaningful sentence